Gemfile

```diff
@@ -31,10 +31,16 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'database_cleaner'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'shoulda-matchers', '~> 3.1'
+  gem 'factory_bot_rails'
+  gem 'ffaker'
+  
 end
 
 group :development do
+  gem 'rspec-rails'
```

Comment:
it should be in `:development, :test` group

Reply:
Or even in `:test` group only.